packages/ref-data-uploaders/txc-uploader/txc_processor.py

```diff
@@ -252,20 +276,36 @@ def get_admin_area_codes(cursor: aurora_data_api.AuroraDataAPICursor, stop_codes
 def insert_into_txc_journey_pattern_table(
     cursor: aurora_data_api.AuroraDataAPICursor, operator_service_id, journey_pattern_info, joined_section_refs
 ):
-    query = "INSERT INTO service_journey_patterns_new (operatorServiceId, destinationDisplay, direction, routeRef, sectionRefs) VALUES (:op_service_id, :destination_display, :direction, :route_ref, :section_refs)"
+    query = "INSERT INTO service_journey_patterns_new (operatorServiceId, destinationDisplay, direction, routeRef, journeyPatternRef, sectionRefs) VALUES (:op_service_id, :destination_display, :direction, :route_ref, :journey_pattern_ref, :section_refs)"
     cursor.execute(
         query,
         {
             "op_service_id": operator_service_id,
             "destination_display": journey_pattern_info["destination_display"],
             "direction": journey_pattern_info["direction"],
             "route_ref": journey_pattern_info["route_ref"],
+            "journey_pattern_ref": journey_pattern_info["journey_pattern_ref"],
             "section_refs": joined_section_refs
         },
     )
     journey_pattern_id = cursor.lastrowid
     return journey_pattern_id
 
+def insert_into_txc_vehicle_journey_table(
+    cursor: aurora_data_api.AuroraDataAPICursor, vehicle_journeys_info
+):
+    values = [
+       {
+            "vehicle_journey_code": vehicle_journey_info["vehicle_journey_code"],
+            "service_ref": vehicle_journey_info["service_ref"],
+            "line_ref": vehicle_journey_info["line_ref"],
+            "journey_pattern_ref": vehicle_journey_info["journey_pattern_ref"],
+        }
+        for vehicle_journey_info in vehicle_journeys_info
```

Comment:
Observed an error "'NoneType' object is not iterable"

Reply:
done